rfc-editor/rfc8794.xml

```diff
@@ -1,32 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE rfc SYSTEM "rfc2629-xhtml.ent">
-<!-- updated by Chris 02/26/20 -->
-
-
-
-<rfc version="3" ipr="trust200902" docName="draft-ietf-cellar-ebml-17"
-     number="8794" submissionType="IETF" category="std" consensus="true"
-     xml:lang="en" xmlns:xi="http://www.w3.org/2001/XInclude"
-     tocInclude="true" symRefs="true">
+<!-- name="GENERATOR" content="github.com/mmarkdown/mmark Mmark Markdown Processor - mmark.miek.nl" -->
+<rfc version="3" ipr="trust200902" docName="draft-ietf-cellar-ebml-18" submissionType="IETF" category="std" xml:lang="en" xmlns:xi="http://www.w3.org/2001/XInclude" tocInclude="true" symRefs="true" consensus="true" number="8794">
```

Comment:
Should this not become `draft-ietf-cellar-ebml-19`, like in the Makefile?

Reply:
No, that's the version we sent them and they edited. Our version will be `draft-ietf-cellar-ebml-19` but it's not generated in the same location.